### PR TITLE
Added Circle CI integration to run Android unit tests

### PR DESCRIPTION
**Pull request by bestander**
A few caveats before accepting:
- Do I need to squash commits?
- Need to set up new Circle CI account connected to FB react-native repo
- After that replace tokens and links to the new ones

Setting up Integration tests should be straight forward next week https://circleci.com/docs/android

### CODE REVIEW

**Comment by mkonicek** [comment]
Looks like this might work: 

[Circle CI](https://circleci.com/gh/facebook/react-native/tree/master)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# React Native [![Build Status](https://travis-ci.org/facebook/react-native.svg?branch=master)](https://travis-ci.org/facebook/react-native)  [![npm version](https://badge.fury.io/js/react-native.svg)](http://badge.fury.io/js/react-native)
+# React Native
+[![Build Status](https://travis-ci.org/facebook/react-native.svg?branch=master)](https://travis-ci.org/facebook/react-native)  [![npm version](https://badge.fury.io/js/react-native.svg)](http://badge.fury.io/js/react-native)
+[![Android Build Status](https://circleci.com/gh/bestander/react-native.svg?style=shield&circle-token=105b18c00e1844eb9300f11adbdae6dda895d2ea)](https://travis-ci.org/facebook/react-native)
```
Comment (mkonicek) [inline README.md]: Might want to leave this out while you're iterating on it.

**Comment by mkonicek** [comment]
This is huge! :rocket: :tada:

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**